buglist-link.js

```diff
@@ -0,0 +1,37 @@
+var bugs = [];
```

Comment:
it could be const and insided getCommonLandings

Reply:
i incorporated your feedback in a new commit